cmake/ActsCompilerOptions.cmake

```diff
@@ -4,7 +4,7 @@ if (NOT CMAKE_BUILD_TYPE)
   message(STATUS "Setting default build type: ${CMAKE_BUILD_TYPE}")
 endif() 
 
-set(ACTS_CXX_FLAGS "-Wall -Wextra -Wpedantic -Wshadow -Wunused-local-typedefs")
+set(ACTS_CXX_FLAGS "-Wall -Wextra -Wpedantic -Wshadow -Wno-unused-local-typedefs")
```

Comment:
Is there a reason we need to disable this warning?

Reply:
It gives false positives at `MultiEigenStepperLoop.hpp:452`, where I have a local struct with typedefs which than is returned from the function...